wgpu/src/backend/wgpu_core.rs

```diff
@@ -411,23 +411,26 @@ fn map_load_op<V: Default>(op: LoadOp<V>) -> (wgc::command::LoadOp, V) {
     }
 }
 
-fn map_pass_channel<V: Copy + Default>(
+fn map_pass_channel<T: TryFrom<V> + Default, V: Copy + Default>(
     ops: Option<&Operations<V>>,
-) -> wgc::command::PassChannel<V> {
+) -> wgc::command::PassChannel<T>
+where
+    <T as std::convert::TryFrom<V>>::Error: std::fmt::Debug,
+{
     match ops {
         Some(&Operations { load, store }) => {
             let (load_op, clear_value) = map_load_op(load);
             wgc::command::PassChannel {
                 load_op: Some(load_op),
                 store_op: Some(map_store_op(store)),
-                clear_value,
+                clear_value: clear_value.try_into().unwrap(),
                 read_only: false,
             }
         }
         None => wgc::command::PassChannel {
             load_op: None,
             store_op: None,
-            clear_value: V::default(),
+            clear_value: T::default(),
```

Comment:
This generic black magic is used to create f32 -> Option<f32> conversion (when needed for depth).

Reply:
Can we have `map_load_op` return `Option<V>` instead?

Reply:
Ah no, that won't work for stencil, although we could make the clear value optional for stencil as well and later default it to 0 if it's not provided (per spec).

Reply:
Then we would need to do needless conversion PassChannel<Option<_>> to PassChannel<_> for stencil in wgpu-core ..., and what we currently have for depth is not nice either.

Reply:
Actually I was wrong as this brings some nice simplifications, so I done it in https://github.com/gfx-rs/wgpu/pull/6753/commits/fd8f07b21ed0abc0cff9e2b429640f4810200b8e.